pkg/test/resourcefixture/testdata/basic/tags/v1beta1/tagstagbinding/tagbindingbasic/create.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
   name: tagstagbinding-${uniqueId}
 spec:
   parentRef:
-    name: project-${uniqueId}
+    # TODO: projectId should work
+    external: //cloudresourcemanager.googleapis.com/projects/${projectNumber}
```

Comment:
I may have lost state on this question, but are we going to make `project-${uniquedId}` work as a follow on or test that it works as a follow on?

Reply:
Yes, I think we should make it work when we have a direct actuation implementation!